src/tribler/core/components/ipv8/eva/exceptions.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from typing import Optional, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from tribler.core.components.ipv8.eva.transfer import Transfer
```

Comment:
Cool, I didn't know that `TYPE_CHECKING` allows to solve cyclic imports when types are needed for type checks only

Reply:
🚀